README.md

```diff
@@ -4,10 +4,33 @@ Welcome to the Paseo Action Submission (PAS) Repository, the central hub for all
 ## Purpose
 The Paseo Action Submission Repository serves as a structured platform for submitting, reviewing, and discussing proposals that impact the Paseo network's technical specifications, governance, features, and improvements. Our aim is to ensure that the Paseo testnet evolves in a way that meets the community's needs and adheres to high standards of quality and innovation.
 
+## PAS workflow
+The individuals involved in creating a PAS are:
+1. The author of the PAS
+2. the Paseo Governance Team (PGT) which are the signers on the sudo multisig
```

Comment:
We called the support and development team, **Paseo Core Team**. Should we add this grup as a third author?

I suspect PGT stands for the Bounty curators correct?

Reply:
I saw "Paseo Governance Team" referenced somewhere else - happy to go with whatever and set a standard.
IMO there are two groups
1. Core team that have sudo access for changing the chain - this group should be in charge of governance of the network and changes etc
2. Curator team that are approving and reviewing payments